integration-tests/verification-flow/range-proofs.test.ts

```diff
@@ -49,92 +49,94 @@ describe('Range proofs verification', () => {
     ).toBeUndefined();
   });
 
-  it('should generate range proofs presentation using number max placeholder', async () => {
-    const wallet: IWallet = await getWallet();
-    const controller = await createVerificationController({
-      wallet,
-    });
-
-    const credentialUrl =
-      'https://creds-testnet.truvera.io/3e824efbead0f72cbe0070923d981762122bf5440cfce513e96b65482ea4934a';
-    const password = '1234';
-    const {data: credential} = await axios.get(
-      `${credentialUrl}?p=${btoa(password)}`,
-    );
-
-    await getCredentialProvider().addCredential(credential);
-
-    await controller.start({
-      template: {
-        boundCheckSnarkKey,
-        signature: null,
-        qr: 'https://creds-testnet.truvera.io/proof/b8acdd33-e8e0-491f-8674-6ba521d9ea84',
-        id: 'b8acdd33-e8e0-491f-8674-6ba521d9ea84',
-        name: 'test data',
-        nonce: '798b999c18bb83f0f9669aac91c68910',
-        created: '2025-01-15T16:34:06.077Z',
-        response_url:
-          'https://api-testnet.truvera.io/proof-requests/b8acdd33-e8e0-491f-8674-6ba521d9ea84/send-presentation',
-        request: {
-          id: 'b8acdd33-e8e0-491f-8674-6ba521d9ea84',
-          input_descriptors: [
-            {
-              id: 'Credential 1',
-              name: 'test data',
-              purpose: 'test',
-              constraints: {
-                fields: [
-                  {
-                    path: ['$.credentialSubject.id'],
-                    optional: false,
-                  },
-                  {
-                    path: ['$.credentialSubject.number'],
-                    filter: {
-                      type: 'number',
-                      exclusiveMinimum: 0,
-                    },
-                    optional: false,
-                    predicate: 'required',
-                  },
-                  {
-                    path: ['$.credentialSchema.id'],
-                    filter: {
-                      const:
-                        'https://schema.truvera.io/TestingCheqdSchema-V1-1736958567072.json',
-                    },
-                  },
-                  {
-                    path: ['$.credentialSubject.name'],
-                    optional: false,
-                  },
-                ],
-              },
-            },
-          ],
-        },
-        type: 'proof-request',
-      },
-    });
-
-    const attributesToReveal = [
-      'credentialSubject.id',
-      'credentialSubject.name',
-      'credentialSubject.date',
-      'credentialSubject.number',
-    ];
-
-    controller.selectedCredentials.set(credential.id, {
-      credential: credential,
-      attributesToReveal,
-    });
-
-    const presentation = await controller.createPresentation();
-
-    expect(
-      presentation.verifiableCredential[0].credentialSubject,
-    ).toBeDefined();
-  });
+  // TODO: The DID for this credential is not available anymore
```

Comment:
why isn't the DID available any more?

Reply:
I'm not sure yet; Agne shared This credential a few weeks ago. I will need to investigate a bit more